cloud-data-protection/src/components/login.tsx

```diff
@@ -0,0 +1,54 @@
+import React, {useEffect, useState} from "react";
+import './login.css';
+import {AuthService} from "../services/authService";
+import {Button, Divider, Input, InputLabel, Typography} from "@material-ui/core";
+import {useSnackbar} from 'notistack';
+import {http} from "../common/http";
+
+const Login = () => {
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+
+    const { enqueueSnackbar } = useSnackbar();
+
+    const authService = new AuthService();
+
+    const handleSubmit = async (e: any) => {
+        e.preventDefault();
+
+        const cancelToken = http.CancelToken.source().token;
+
+        const input = { email, password };
+
+        await authService.login(input, cancelToken)
+            .then(() => onSuccess())
+            .catch((e: any) => onError(e))
+            .finally(() => onFinish());
+    }
+
+    const onSuccess = () => {
+        setEmail('');
+    }
+
+    const onError = (e: any) => {
+        enqueueSnackbar(e, { autoHideDuration: 6000 });
+    }
+
+    const onFinish = () => {
+        setPassword('');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
        setPassword("");
```